ledger/src/blockstore_db.rs

```diff
@@ -127,7 +127,7 @@ pub type Result<T> = std::result::Result<T, BlockstoreError>;
 
 impl std::fmt::Display for BlockstoreError {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        write!(f, "blockstore error")
+        write!(f, "blockstore error: {}", self)
```

Comment:
does this actually work, I thought `"{}"` would call fmt. Maybe it should be `{:?}` to print debug?